app/models/concerns/pick_up_order.rb

```diff
@@ -0,0 +1,89 @@
+class PickUpOrder < ActiveRecord::Base
+  # rubocop:disable Metrics/BlockLength
```

Comment:
That sounds like a lot of effort just to avoid a "long" block that's perfectly clear and maintainable as-is. But again, this is just my 2c, not worth holding anything up over it. I do like the idea of a config repo for this kind of thing since the standards should be enforced across all projects.